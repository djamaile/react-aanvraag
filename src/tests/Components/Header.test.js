import {shallow} from 'enzyme';
import React from 'react';
import toJSON from 'enzyme-to-json';
import Header from '../../components/AppHeader';

test('Header moet goed tevoorschijn komen', () => {
    const wrapper = shallow(<Header/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
});
