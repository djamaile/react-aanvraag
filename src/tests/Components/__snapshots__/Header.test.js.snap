// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header moet goed tevoorschijn komen 1`] = `
<header>
  <h1>
    Aanvraag Applicatie
  </h1>
  <NavLink
    activeClassName="isActive"
    aria-current="page"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="isActive"
    aria-current="page"
    to="/aanvraag"
  >
    Aanvraag
  </NavLink>
</header>
`;
